docs/dev/code_review.md

```diff
@@ -0,0 +1,121 @@
+# Code Reviews 
+
+Reviewing code is like proof reading a paper, it is essential to ensure the following: 
+
+- code is user readable
+- enforces consistency
+- helps encourage collaboration
+- code has a proper structure
+
+!!! Note
+    Code review is not meant for debugging or finding bugs
+
+Anyone can code review even if you don't program in the native language, which for pyDARN is python. 
+Code reviewing is also an excellent learning experience which helps you better understand the software and how to program in general. 
+
+Code reviews should not take more than an hour and Pull Requests should be less than 400 lines.
+However, these restrictions can be challenging for new features and major changes. 
+If this is the case, then you can code review parts of a file or specific files to reduce your time spent. 
+
+## How to start a Code Review 
+
+1. Pick a Pull Request to code review, which can be found [here](https://github.com/SuperDARN/pydarn/pulls)
+2. Click on `files changed` at the top of the PR conversation below the title on the right side 
+3. Now review the changes that occurred in this PR - avoid commenting on other lines not included in the changes as this is `scope creep`.
+4. If you want to make a comment hover the cursor over the line numbers on the left side and a **+** button will appear. Click it!
+5. Now write any feedback or questions in the box and hit `Start a review` on the bottom right in the pop up window. 
+    
+    !!! Warning
+        Do not make several single comments as this can cause a lot of email notifications
+
+6. If you want to change a line in the code use a [suggestion](https://haacked.com/archive/2019/06/03/suggested-changes/)
+7. If you wish to comment on several consecutive lines hold **+** down and drag the cursor down to select multiple lines
+8. Once you are done, click on `Finish your changes` or `Review changes` (if no comments made) then comment on any general feedback in the text box
```

Comment:
Here is where I would suggest potentially including what type of review was done, a Formatting Review and/or a Testing Review.